workflows/rnaseq-ref-index/build-index.nf

```diff
@@ -32,12 +37,14 @@ process salmon_index_no_sa{
     salmon index \
       -t ${reference} \
       -i ${index_base}_k${kmer} \
-      -k ${kmer}
+      -k ${kmer} \
+      -p ${task.cpus} \
+      --gencode
```

Comment:
Do we need this flag? I don't think any of our fasta files use gencode formatting.